### ISSUE TITLE
Question - Is there a way to get the temperature of GPU and CPU?

### ISSUE THREAD

**Issue by JRQuimora** [Abdulazeezvp/windows_system_info#2]
Is there a way to get the temperature of GPU and CPU? maybe realtime usage also? there is little to no documentation about this so I'm struggling to know what I can do with this plugin.

**Comment by Abdulazeezvp** [comment]
No, as of now, it's not possible to find the temperature of the CPU or GPU using this library. Yes, it's possible to fetch the temperature of the CPU from a WMI object, but it requires administrative permission to fetch details from MSAcpi_ThermalZoneTemperature. While we execute a command from Dart, it will return a null or error string in this case (in the default case).
Thanks for the idea to improve this package; I will try to add similar features where possible, and if you could contribute anything, please do it.

[tool to create wmi quries](https://www.microsoft.com/en-us/download/details.aspx?id=8572) by comparing code in this library you can do more to it

https://github.com/Abdulazeezvp/windows_system_info/blob/7004e8d75b6eba1feeadeba1881b2ffa64dadeff/lib/src/general_helper.dart#L9

**Comment by Abdulazeezvp** [comment]
Hi, I’ve added some functions to this feature request. Could you please help me with testing it? Sorry for being late in adding this feature. I’m still not able to fetch CPU temperature on Intel laptops, this is just an experiment for now. The changes are available in the cpu_temp branch.
